internal/testhelpers/suite/suite.go

```diff
@@ -0,0 +1,50 @@
+package suite
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/ActiveState/cli/internal/errs"
+	"github.com/stretchr/testify/require"
+	testify "github.com/stretchr/testify/suite"
+)
+
+type tSuite interface {
+	testify.TestingSuite
+}
+
+type Suite struct {
+	testify.Suite
+}
+
+func (suite *Suite) NoError(err error, msgAndArgs ...interface{}) {
+	if err == nil {
+		return
+	}
+
+	errMsg := fmt.Sprintf("All error messages: %s", errs.JoinMessage(err))
+	msgAndArgs = append(msgAndArgs, errMsg)
+	suite.Suite.NoError(err, msgAndArgs...)
+}
+
+func (suite *Suite) Require() *Assertions {
+	return &Assertions{suite.Suite.Require()}
+}
+
+type Assertions struct {
+	*require.Assertions
+}
+
+func (a *Assertions) NoError(err error, msgAndArgs ...interface{}) {
+	if err == nil {
+		return
+	}
+
+	errMsg := fmt.Sprintf("All error messages: %s", errs.JoinMessage(err))
+	msgAndArgs = append(msgAndArgs, errMsg)
+	a.Assertions.NoError(err, msgAndArgs...)
```

Comment:
Same as with the other one; should just implement the NoError logic here and avoid calling out to testify at all, so we don't have duplicated errors. And then you can probably share logic also.